_posts/2021-06-07-changelog-148.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #148"
+date: 2021-06-07 17:00:00 +0300
+---
+
+
+## New Features
+
+* Refactoring improvements:
+
+    * [#7255] Add support for default values to suggested change signature refactoring (by [@Kobzol])
+
+    * [#7135] Add suggested refactoring support for changing function signatures (by [@Kobzol])
+
+    * [#6957] Create target file when moving items (<kbd>F6</kbd> or `Refactor | Move`) if it doesn't exist
+
+* [#7239] Check for missing input file when using redirect input in `Cargo command` run configuration
+
+* [#6825] `match` postfix template now generates all missing match arms by default (by [@Kobzol])
+
+* [#7237] Update experimental code completion ranking for Rust based on machine learning. The ranking model is disabled
+  by default for now. You can enable it via `Editor | General | Code Completion | Machine Learning-Assisted Completion`
+  [settings](https://www.jetbrains.com/help/idea/auto-completing-code.html#ml_completion).
+
+* [#7259] Support 2021.2 EAPs
+
+## Fixes
+
+* Windows Subsystem for Linux (WSL) fixes. It can be enabled using `org.rust.wsl`
+  [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features):
+
+    * [#7284] Fix fetching of cfg-options
+
+    * [#7268] Fix toolchain validation
+
+* [#7271] Don't throw exceptions on illegal paths in `PATH` environment variable during toolchain detection
+
+* [#7253] `Add remaining arms` intention now prints enum names and variants correctly if they are escaped rust keywords
+  (by [@shocoman])
+
+* [#7248] Try to recover corrupted stdlib dependencies when `org.rust.cargo.fetch.actual.stdlib.metadata`
+  [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features) is
+  enabled
+
+* [#7244] Don't annotate `async` as a reserved keyword in attributes
+
+* [#7236] Change the completion context of path string literals to the containing workspace root (by [@Kobzol])
+
+* [#6673] Complete `continue` and `break` statements inside expression contexts in loops (by [@Kobzol])
+
+## Internal Improvements
+
+* [#7274] Improve cancellation of outdated CI runs
+
+* [#7265] RUN: Add Cargo commands logging
+
+* [#7105] Fix some type annotations in python pretty printers (by [@lancelote])
+
+Full set of changes can be found [here](https://github.com/intellij-rust/intellij-rust/milestone/56?closed=1)
+
+[@Kobzol]: https://github.com/Kobzol
+[@lancelote]: https://github.com/lancelote
```

Comment:
Let's exclude `lancelote` name from the changelog since he is a maintainer. I've already added his nick to maintainer list in `changelog.py`. Don't forget to drop the corresponding mentions